File: src/main/java/com/cmpe172/orderup/model/CartItem.java
package com.cmpe172.orderup.model;

public class CartItem {
	private String name;
	private double price;
	private String uuid;
	
	public CartItem() {
		super();
	}

	public CartItem(String name, double price, String uuid) {
		super();
		this.name = name;
		this.price = price;
		this.uuid = uuid;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public double getPrice() {
		return price;
	}

	public void setPrice(double price) {
		this.price = price;
	}

	public String getUuid() {
		return uuid;
	}

	public void setUuid(String uuid) {
		this.uuid = uuid;
	}
}

File: frontend/lib/model/cart_item.dart
import 'package:food_ordering/model/menu_item.dart';
import 'package:uuid/uuid.dart';

class CartItem {
  final String name;
  final double price;
  final String uuid;

  CartItem({this.name, this.price, this.uuid});

  CartItem.fromMenuItem(MenuItem menuItem)
      : name = menuItem.name,
        price = menuItem.price,
        uuid = Uuid().v1();

  CartItem.fromMap(Map map)
      : name = map['name'],
        price = map['price'],
        uuid = map['uuid'];

  Map toMap() => {
        'name': name,
        'price': price,
        'uuid': uuid,
      };
}
